### ISSUE TITLE
404 errors problem with multilangual permalink pages

### ISSUE THREAD

**Issue by Ishanjn** [not-only-code/qtranslate-slug#133]
I am suffering an unexpected issue with “Plugin: Qtranslate Slug”; Some of my pages will automatically redirected to 404 error page. I tried a lot of this in slug but the issue is not resolved. This error mostly occurred on child pages.

In terms of solving the problem I need to remove database field value which holding slug value. After remove column values page working fine.

Can anyone please help me to override this problem; it’s very urgent? I also found an issue on a forum ” https://wordpress.org/support/topic/404-errors-and-problem-with-child-pages/ ”

I to dig for solutions for that and I found when I requested for a page with language slug plugin hook for ‘request’, getting post_type = ‘attachment’. But this is not right because I am requesting for ‘page’.
I tried to dump post_type under `filter_request` function.

`add_filter(‘request’, array(&$this, ‘filter_request’));`

I think this issue related with my problem - https://github.com/not-only-code/qtranslate-slug/issues/130

But this issue is not only with my old pages also same problem exist in newly created pages.

**Comment by LC43** [comment]
Hi @Ishanjn , did you see my reply o wordpress.org?

could you try first my suggestion?  thanks

**Comment by Ishanjn** [comment]
Yes, its work after build parent page qts-slug and then rebuild child page qts-slug. 
Thank you very much.

**Comment by marcolettieri** [comment]
@lc43 you have finished the work on?

**Comment by webplease** [comment]
same problem here, only with RU (cyrillic) language ... have u solved?

**Comment by netyco** [comment]
i have the samme problem, please help :(
